OpenRA.Mods.Common/AI/States/GroundStates.cs

```diff
@@ -140,9 +147,11 @@ public void Tick(Squad owner)
 				}
 			}
 
+			// Switch target during fight
```

Comment:
Ah, thought the comment was referring to possible different results of the individual `FindClosesetEnemy` calls in the loop.